examples/timeseries_peak_finding.py

```diff
@@ -0,0 +1,78 @@
+"""
+==============================
+Find Peaks in SunPy TimeSeries
+==============================
+
+This example illustrates how to find minimum or maximum peaks in a TimeSeries.
+"""
+
+##############################################################################
+# Start by importing the necessary modules.
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import sunpy.data.sample
+from sunpy.timeseries import TimeSeries
+
+##############################################################################
+# We will now create a TimeSeries object from an observational data source and 
+# do analysis on its truncated version.
+
+ts_noaa_ind = sunpy.timeseries.TimeSeries(
+    sunpy.data.sample.NOAAINDICES_TIMESERIES, source='NOAAIndices')
+my_timeseries = ts_noaa_ind.truncate(0,100)
+my_timeseries.peek()
+
+##############################################################################
+# Now we take the column 'sunspot SWO' of this TimeSeries and try to find it's
+# peaks. For peak finding, we first decide a DELTA value which controls how
```

Comment:
Can you explain your choice of the `DELTA` value?

Reply:
Umm, it is by observation only, somewhat of the order of number of sunspots on y axis.